pkg/vadmin/start_db_vc.go

```diff
@@ -1,33 +1,66 @@
-/*
- (c) Copyright [2021-2023] Open Text.
- Licensed under the Apache License, Version 2.0 (the "License");
- You may not use this file except in compliance with the License.
- You may obtain a copy of the License at
+// /*
```

Comment:
Can we revert back to the old comment style? I would like all files to have the same header comment.